CriteoPublisherSdk/Sources/Configuration/CR_IntegrationRegistry.m

```diff
@@ -45,6 +47,12 @@ - (void)declare:(CR_IntegrationType)integrationType {
 }
 
 - (NSNumber *)profileId {
+  if (self.isMoPubMediationPresent) {
+    return @(CR_IntegrationMopubMediation);
+  } else if (self.isAdMobMediationPresent) {
```

Comment:
Does this have a strong requirement of mutually exclusive?
I'm thinking of the test app, that could help not to be.

Reply:
What do you propose when both are detected ?

Reply:
Well, I doubt we want to introduce bit mask logic there to declare both at the same time.
What bothers me is that in such situation Google "wins".
On test app we will override the value as you did on Android.
Maybe if both are detected we should report `Fallback` instead?

Reply:
Ok to use fallback then, it makes more sense